composer.json

```diff
@@ -52,9 +52,9 @@
         "phpstan/phpstan-webmozart-assert": "^1.1",
         "phpunit/phpunit": "^8.5",
         "polishsymfonycommunity/symfony-mocker-container": "^1.0",
-        "rector/rector": "^0.14.6",
         "stripe/stripe-php": "^6.43",
         "sylius-labs/coding-standard": "^4.0",
+        "sylius/sylius-rector": "0.11.x-dev",
```

Comment:
Why do we have `0.11.x-dev` version there?

Reply:
I think both `0.11.x-dev` and `dev-main` are correct as `0.11-dev` is set as an alias

Reply:
My point was that SyliusRector has no released version yet, so why is it targeting 0.11 instead of 0.1 (or possibly 1.0)?

Reply:
I relied on [this](https://github.com/Sylius/SyliusRector/blob/main/composer.json#L48). If it's wrong I'll change it